src/servers/src/http/result/greptime_manage_resp.rs

```diff
@@ -68,6 +78,8 @@ pub enum ManageResult {
 pub struct PipelineOutput {
     name: String,
     version: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pipeline: Option<Value>,
```

Comment:
`pipeline` can be output as pretty YAML and it's easy to read, maybe `Option<String>` will be better.